src/psyclone/gocean1p0.py

```diff
@@ -360,6 +361,7 @@ def const_loop_bounds(self, obj):
         self._const_loop_bounds = obj
 
 
+# pylint: disable=too-many-instance-attributes
 class GOLoop(Loop):
```

Comment:
Please add information on the interface to the constructor to the class docstring.